sections/05-adc-data-publishing.qmd

```diff
@@ -196,18 +236,32 @@ If you know that you will need to use this process for more than one dataset, we
+### Data Organization
 
-When you're data are associated with your metadata submission, they will appear in the data section at the top of the page when you go to edit your dataset. Choose which file you would like to begin editing by selecting the "Describe" button to the right of the file name.
+The data archival process is much smoother when users already have a well structured and organized file management system. As we are at the end of the research life cycle for many users, we understand that you may not already have a well organized file system. In cases such as these, we recommend organizing your data as much as possible ***before*** upload.
 
-![](../images/submissions-form-data-file.png)
+Take a look at this recently made [data portal](https://arcticdata.io/catalog/portals/AIS). Originally, this user had only submitted the last listed dataset. That dataset contains over 700 rasters and shapefiles depicting vessel paths in the North Pacific at many different resolutions. Upon review, the submitter determined that it would be best if other users could download the data at specific resolution scales in addition to the whole collection of data. By creating separate datasets at the resolution scale, this user was able to organize their data in a structured way that makes it easy for future researchers to use their data. Furthermore, the user was able to house their related datasets under one roof using our portal system.
 
-Once there, you will see the following screen. In the Overview section, we recommend ***not*** editing the file name, and instead add a descriptive overview of the file. Once done, click the Attributes tab.
+::: callout-note
+To find out more about portals, visit <https://arcticdata.io/data-portals/>. You can view our catalog of portals [here](https://arcticdata.io/catalog/portals). We recommend reviewing the Distributed Biological Observatory's [portal](https://arcticdata.io/catalog/portals/DBO) to see what a more fleshed out portal can look like.
+:::
 
-![](../images/editor-18-socialsci.png)
+![](../images/kapsar-portal.png)
+
+This example also illustrates that users can create multiple datasets for a single NSF award. It's understandable that you may think you have to submit all your data for a given award under one dataset. However, we encourage you to be flexible in how you approach archiving and organizing your data with us as this user has been.
+
+When uploading your data files using SFTP, either through the command line or a GUI, we recommend creating folders with the same name as your dataset's title so that it is easier to associate the data with their associated metadata submission.
```

Comment:
I feel mixed about whether to keep this tip or not. maybe add it to the callout below

Reply:
Info's been moved into the below callout